.cirrus.yml

```diff
@@ -52,6 +52,7 @@ build_task:
     - npm install
     - npm run install-dep-full
   script:
+    - git fetch
```

Comment:
If we don't run `git fetch`, the master branch does not exist and the scanner can't compare the content of the dev branch (pull request) to the mater branch